### PR TITLE
Update Nutmeg entry: 2FA is supported via TOTP app.

### PR DESCRIPTION
**Pull request by Flameeyes**

### CODE REVIEW

**Review by stephengroat — DISMISSED** [review]
thanks for the pr @Flameeyes

File: _data/investing.yml [hunk 1]
```diff
@@ -98,8 +98,11 @@ websites:
     - name: Nutmeg
       url: https://www.nutmeg.com/
       twitter: thenutmegteam
+      facebook: TheNutmegTeam
```
Comment (mxxcon) [inline _data/investing.yml]: 'twitter' should NOT be present when `tfa: true`.
'facebook' should NOT be present when `tfa: true`.

**Review by mxxcon — CHANGES_REQUESTED** [review]

**Comment by Flameeyes** [comment]
Updated, PTAL.

**Comment by Flameeyes** [comment]
Ping?

**Review by mxxcon — APPROVED** [review]

**Comment by Flameeyes** [comment]
Hi folks, any update on this one? :)

**Review by Jawshy — APPROVED** [review]
Looks good. Thanks for the pull request, @Flameeyes. 

For any future pull requests I'd recommend creating a [new branch](https://help.github.com/articles/creating-and-deleting-branches-within-your-repository/) on your fork.

**Pull request merged**